library-best-practices/JLBP-19.md

```diff
@@ -0,0 +1,23 @@
+[JLBP-19] Place each package in a single jar
```

Comment:
This sounds like it could mean no two packages can go into the same jar.

What about this?

```suggestion
[JLBP-19] Maven artifacts should not overlap
```

Reply:
No, overlap is not as direct and we already use that to mean soething different. Other suggestions?

Reply:
I think adding "only" changes the interpretation to avoid @netdpb 's confusion: "Place each package in only one jar."

Although, I think saying "one jar" is actually misleading, because that sounds like you can't even have the same package in multiple minor versions (which are technically different jars).

It would really be ideal if there was a single term for artifact ID + group ID - I struggled with this in other JLBPs.

Maybe we can use a deliberately vague term and be more specific in the extended description: "Place each package in only one library."

Reply:
Perhaps " "Place each package in only one module"?